dll/win32/ws2_32/src/startup.c

```diff
@@ -74,6 +102,24 @@ WSACleanup(VOID)
         /* Decrement process reference count and check if it's zero */
         if (!(RefCount = InterlockedDecrement(&Process->RefCount)))
         {
+            /* Be sure no other thread is using WSAThread */
+            WsCreateWSAThreadSynchronization();
+
+            /* Send message to thread to shutdown */
+	    WSAThreadCallId = 0;
+            SetEvent(gWSAThreadEvent);
```

Comment:
There is a unnecessary blank indentation space (see 110 line).

Reply:
This was because of tabs instead of spaces, currently replacing them with spaces...